docs/examples/install_tendermint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# get and unpack golang
+curl -O https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz
+tar -xvf go1.9.2.linux-amd64.tar.gz
+
+apt install make
```

Comment:
what if no `apt`? what about `yum`?

Reply:
see previous comment. it's not meant a multi-OS check-all-the-things scripts, but rather as a quick and dirty get-started-ASAP-on-Digital-OCean